src/main/java/org/broadinstitute/hellbender/utils/help/HelpConstants.java

```diff
@@ -8,4 +8,22 @@ public static String forumPost(String post) {
         return GATK_FORUM_URL + post;
     }
 
+    /**
+     * Definition of the group names / categories of tools.
+     * The names get parsed to make supercategories in the doc index,
+     * so be careful when making big changes -- see GATKDoclet.java toMap()
```

Comment:
BTW, there is a [doc](https://github.com/broadinstitute/barclay/wiki/Using-the-Documentation-Generation-utilities-in-Barclay) (under construction) in the Barclay repo that describes some of this.